sensors/sensor.py

```diff
@@ -0,0 +1,67 @@
+import time
+
+try:
+    import pigpio
+except ImportError:
```

Comment:
Sadly `pigpio` is only available when working directly on the Pi as it's part of the `i2c-dev-tools` Deb package, so without this catch, it's impossible to run tests.

Reply:
You can use pigpio to communicate remotely with a pi running pigpiod (the pigpio daemon). 

I've got pigpio on my personal laptop, and can run our code base remotely. Ill I needed to do was 'pip3 install pigpio' on my laptop. Once I did that, you tell pigpio to connect remotely, then I was able to run the python script that communicates with the sensor

Reply:
To support my last comment, take a peek at line 17. I did all debugging remotely off my laptop, 'pi3.local' is the hostname of the pi. 

That is how the pigpio module knows which host to connect to. As long as the host is a pi running pigpiod (the daemon), everything should run smoothly and there should be no problem with importing as long as the device running our script has pigpio installed